src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Main {
+    
+    public static void main(String[] args) {
+
+        Figure[] figures = new Figure[6];
```

Comment:
Consider using a constant for the size of the `figures` array to improve readability and maintainability. For example, `private static final int FIGURE_COUNT = 6;`